gateway/api/signup/signup.go

```diff
@@ -74,6 +74,14 @@ func Post(c *gin.Context) {
 		c.JSON(http.StatusInternalServerError, gin.H{"message": "failed creating organization"})
 		return
 	}
+
+	_, err = models.CreateDefaultRunbookConfiguration(models.DB, org.ID)
+	if err != nil {
+		log.Errorf("failed creating default runbook configuration, err=%v", err)
+		c.JSON(http.StatusInternalServerError, gin.H{"message": "failed creating default runbook configuration"})
+		return
+	}
```

Comment:
idk if we should fail the signup if the runbook, for some reason, fails 🤔 ? @EmanuelJr WDYT?

Reply:
You could call this `CreateDefaultRunbookConfiguration` on the `CreateOrganization`.
There is the logic about the plugins already, so it would not be a big problem.